control-plane-operator/controllers/hostedcontrolplane/kcm/deployment.go

```diff
@@ -348,7 +350,8 @@ func kcmArgs(p *KubeControllerManagerParams) []string {
 		"--kube-api-qps=150",
 		"--leader-elect-resource-lock=configmapsleases",
 		"--leader-elect=true",
-		"--leader-elect-retry-period=3s",
+		fmt.Sprintf("--leader-elect-renew-deadline duration=%s", config.KCMRecommendedRenewDeadline),
```

Comment:
```suggestion
		fmt.Sprintf("--leader-elect-renew-deadline-duration=%s", config.KCMRecommendedRenewDeadline),
```
I believe you are missing a `-` here

Reply:
Nice catch. Thank you for the review!